app/presenters/tree_builder_alert_profile_obj.rb

```diff
@@ -0,0 +1,80 @@
+class TreeBuilderAlertProfileObj < TreeBuilder
+  def initialize(name, type, sandbox, build = true, assign = nil)
+    @assign = {
+      :new     => assign[:new],
+      :current => assign[:current]
+    }
+
+    build = false unless @assign[:new][:assign_to]
+    super(name, type, sandbox, build)
+  end
+
+  def override(node, object, _pid, _options)
+    identifier = (object.name.presence || object.description)
+    if object.kind_of?(MiddlewareServer)
+      identifier += "-" + object.hostname
+    end
+
+    if @assign[:new][:assign_to].ends_with?("-tags")
+      node[:icon] = "fa fa-tag"
+    elsif @assign[:new][:assign_to] == "ext_management_system"
+      node[:image] = "svg/vendor-#{object.image_name}.svg"
+    elsif @assign[:new][:assign_to] == "resource_pool"
+      node[:icon] = "pficon-resource_pool"
+    elsif @assign[:new][:assign_to] == "tenant"
+      node[:image] = ActionController::Base.helpers.image_path("100/tenant.png")
+    else
+      node[:image] = ActionController::Base.helpers.image_path("100/#{@assign[:new][:assign_to]}.png")
```

Comment:
This PNG is copied from the old code, but yes it should be converted. Is there any reason why are defining so many stuff in the `override` method?